infrastructure/datasources.tf

```diff
@@ -0,0 +1,9 @@
+data "azuread_client_config" "current" {}
+
+data "azurerm_role_definition" "aks_cluster_admin" {
```

Comment:
Can we put a comment on what these magic strings equate to for future developers?